frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/olm.ts

```diff
@@ -0,0 +1 @@
+export const installSucess = 'svg[fill="#3e8635"]';
```

Comment:
This is a little vauge?  Is there an ID we can use, or as a fall back, a css selector?

Reply:
A css selector is there but we will need that color code to verify that it''s green we could use `.co-operator-install-page__pkg-indicator > :nth-child(2) > svg[fill="#3e8635"]`.

Reply:
ok

Reply:
I don't think we should be validating specific colors. We should have data attributes for success/error icons and check for those.